lldb/include/lldb/Target/Target.h

```diff
@@ -1433,7 +1444,7 @@ class Target : public std::enable_shared_from_this<Target>,
 
   private:
     StringList m_commands;
-    // Use CreateStopHook to make a new empty stop hook. The GetCommandPointer
+    // Use CreateStopHook to make a new empty stop hook. The SetActionFromString
```

Comment:
This wasn't grammatical originally, but...
Maybe:

Use SetActionFromString to fill it with commands